packages/tpetra/core/CMakeLists.txt

```diff
@@ -14,6 +14,14 @@ TRIBITS_ADD_EXPLICIT_INSTANTIATION_OPTION()
 # template instantiation (ETI).
 TRIBITS_ADD_ETI_SUPPORT()
 
+# adding macro for deprecated code (templated L0 and G0, and dynamic allocation)
+TRIBITS_ADD_OPTION_AND_DEFINE(
+  Tpetra_ENABLE_DEPRECATED_CODE
+  TPETRA_ENABLE_DEPRECATED_CODE
+  "Disable Tpetra deprecated code (templated ordinal types and dynamic graph/matrix allocation)"
+  Tpetra_ENABLE_DEPRECATED_CODE_DEFAULT
```

Comment:
I think you need to dereference the default value here.  Do this:
```
${Tpetra_ENABLE_DEPRECATED_CODE_DEFAULT}
```

Reply:
You're right, I did need that for the default.